torch_geometric/datasets/lrgb.py

```diff
@@ -0,0 +1,280 @@
+                            Shape of y : [1, 10] for Peptides-func,  or
+                                         [1, 11] for Peptides-struct
+                        """
+                        x = graph[0]
+                        edge_attr = graph[1]
+                        edge_index = graph[2]
+                        y = graph[3]
+
+                    if self.name == 'coco-sp':
+                        for i, label in enumerate(y):
+                            y[i] = label_map[label.item()]
+
+                    data = Data(x=x, edge_index=edge_index,
+                                edge_attr=edge_attr, y=y)
+
+                    if self.pre_filter is not None and not self.pre_filter(
+                            data):
+                        continue
+
+                    if self.pre_transform is not None:
+                        data = self.pre_transform(data)
+
+                    data_list.append(data)
+
+                torch.save(self.collate(data_list),
+                           osp.join(self.processed_dir, f'{split}.pt'))
+
+    def label_remap_coco(self):
+        # Util function for name 'COCO-SP'
+        # to remap the labels as the original label idxs are not contiguous
+        original_label_idx = [
+            0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 14, 15, 16, 17, 18, 19,
+            20, 21, 22, 23, 24, 25, 27, 28, 31, 32, 33, 34, 35, 36, 37, 38, 39,
+            40, 41, 42, 43, 44, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57,
+            58, 59, 60, 61, 62, 63, 64, 65, 67, 70, 72, 73, 74, 75, 76, 77, 78,
+            79, 80, 81, 82, 84, 85, 86, 87, 88, 89, 90
+        ]
+
+        label_map = {}
+        for i, key in enumerate(original_label_idx):
+            label_map[key] = i
+
+        return label_map
+
+    def process_pcqm_contact(self):
+        for split in ['train', 'val', 'test']:
+            with open(osp.join(self.raw_dir, f'{split}.pt'), 'rb') as f:
+                graphs = torch.load(f)
+
+            data_list = []
+            for graph in tqdm(graphs, desc=f'Processing {split} dataset'):
+                """
+                PCQM-Contact
+                Each `graph` is a tuple (x, edge_attr, edge_index,
+                                        edge_index_labeled, edge_label)
+                    Shape of x : [num_nodes, 9]
+                    Shape of edge_attr : [num_edges, 3]
+                    Shape of edge_index : [2, num_edges]
+                    Shape of edge_index_labeled: [2, num_labeled_edges]
+                    Shape of edge_label : [num_labeled_edges]
+
+                    where,
+                    num_labeled_edges are negative edges and link pred labels,
+                    https://github.com/vijaydwivedi75/lrgb/blob/main/graphgps/loader/dataset/pcqm4mv2_contact.py#L192
+                """
+                x = graph[0]
+                edge_attr = graph[1]
+                edge_index = graph[2]
+                edge_index_labeled = graph[3]
```

Comment:
[naming] Use `edge_label_index` to be consistent with data split by `torch_geometric.transforms.RandomLinkSplit`.

Reply:
Renamed `edge_index_labeled` to `edge_label_index`.